### ISSUE TITLE
Release MotionStreamer model checkpoint on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [zju3dv/MotionStreamer#28]
Hi @Li-xingXiao 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2503.15451.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

It's fantastic to see the processed datasets (`272-dim HumanML3D`, `272-dim BABEL`, `272-dim BABEL Streaming`) and several model components (`Evaluator_272`, `Causal_TAE`, and `t2m_model`) already hosted on https://huggingface.co/! Thank you for making these openly available.

I noticed that while the training code for the main MotionStreamer model is provided, the pre-trained checkpoint for the MotionStreamer model itself doesn't seem to have a direct link on the Hugging Face Hub yet.

Would you be interested in hosting the MotionStreamer model checkpoint on https://huggingface.co/models as well?
Hosting it on Hugging Face will give it more visibility and enable better discoverability. We can add tags in the model card so that people find the model easier, and link it directly to the paper page.

If you're interested, here's a guide: [https://huggingface.co/docs/hub/models-uploading](https://huggingface.co/docs/hub/models-uploading). If it's a custom PyTorch model, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class, which adds `from_pretrained` and `push_to_hub` methods, making it easy for others to download and use your model.

Let me know if you're interested or need any guidance!

Kind regards,

Niels
